lib/stdlib/test/erl_lint_SUITE.erl

```diff
@@ -4448,6 +4449,46 @@ inline_nifs(Config) ->
            {warnings,[{2,erl_lint,nif_inline}]}}],
     [] = run(Config, Ts).
 
+compile_options(Config) ->
```

Comment:
Most of the test cases in this module test compiler options. Please rename it to a more descriptive name.